### PR TITLE
104017 rest of columns in post endepoint vol2

### PR DESCRIPTION
**Pull request by janid1967**
[AB#104017](https://statoildeveloper.visualstudio.com/b6d97460-2e53-47ae-aad4-b41396d90828/_workitems/edit/104017)

### CODE REVIEW

File: src/tests/Equinor.ProCoSys.Completion.Command.Tests/EventHandlers/DomainEvents/PunchItemEvents/PunchItemCategoryUpdatedEventHandlerTests.cs [hunk 1]
```diff
@@ -49,18 +49,45 @@ await _publishEndpointMock.Received(1)
...
+        // Arrange
+        FillOptionalProperties(_domainEvent.PunchItem);
+
+        // Act
+        await _dut.Handle(_domainEvent, default);
+
+        // Assert
+        Assert.IsNotNull(_publishedIntegrationEvent);
+        Assert.AreEqual($"Punch item category changed to {_domainEvent.PunchItem.Category}", _publishedIntegrationEvent.DisplayName);
+        Assert.AreEqual(_domainEvent.PunchItem.Guid, _publishedIntegrationEvent.Guid);
```
Comment (jsoi-eq) [inline src/tests/Equinor.ProCoSys.Completion.Command.Tests/EventHandlers/DomainEvents/PunchItemEvents/PunchItemCategoryUpdatedEventHandlerTests.cs:84]: Any reason not to Assert on ModifiedAt/By in this test, but doing so in the test above?
I see this is a repeating pattern in several unit tests classes.

**Review by jsoi-eq — COMMENTED** [review]
Awaiting feedback.

File: src/Equinor.ProCoSys.Completion.Command/EventHandlers/DomainEvents/PunchItemEvents/IntegrationEvents/PunchItemCreatedIntegrationEvent.cs [hunk 2]
```diff
@@ -28,14 +47,33 @@ public record PunchItemCreatedIntegrationEvent
         punchItemCreatedEvent.PunchItem.Project.Guid,
         punchItemCreatedEvent.PunchItem.Project.Name,
         punchItemCreatedEvent.PunchItem.Project.Description,
+        punchItemCreatedEvent.PunchItem.CheckListGuid,
+        punchItemCreatedEvent.PunchItem.Category.ToString(),
```
[the remark this answers is not in the document]
Comment (janid1967) [reply src/Equinor.ProCoSys.Completion.Command/EventHandlers/DomainEvents/PunchItemEvents/IntegrationEvents/PunchItemCreatedIntegrationEvent.cs:51]: Minimum ot no risk

File: src/tests/Equinor.ProCoSys.Completion.Command.Tests/EventHandlers/DomainEvents/PunchItemEvents/PunchItemCategoryUpdatedEventHandlerTests.cs (hunk 1 above)
Comment (janid1967) [reply src/tests/Equinor.ProCoSys.Completion.Command.Tests/EventHandlers/DomainEvents/PunchItemEvents/PunchItemCategoryUpdatedEventHandlerTests.cs:83]: Uniformed tests with AssertModified()

**Review by jsoi-eq — APPROVED** [review]

**Pull request merged**